apps/mobile/app/(authenticated)/account/[id]/signAndSend/ioPreview.tsx

```diff
@@ -72,16 +84,129 @@ export default function IOPreview() {
     setAddOutputModalVisible(false)
   }
 
+  const { width, height } = useWindowDimensions()
+  const GRAPH_HEIGHT = height - 100
+  const GRAPH_WIDTH = width
+
+  const canvasSize = { width: GRAPH_WIDTH * 1.5, height: GRAPH_HEIGHT }
+  const sankeyWidth = canvasSize.width
+  const sankeyHeight = canvasSize.height - 200
+
+  const translateX = useSharedValue(-width * 0.4)
+  const translateY = useSharedValue(0)
+  const savedTranslateX = useSharedValue(-width * 0.4)
+  const savedTranslateY = useSharedValue(0)
+
+  const panGesture = Gesture.Pan()
```

Comment:
I wonder if we would have the sankey diagram in multiple pages, this code should be moved inside the component or a wrapper component in the future

Reply:
the SSUtxoFlow is the diagram, we can just reuse it if we want to show the sankey diagram. The gesture IMHO should be outside of the SSUtxoFlow component. 

yeah the naming is a bit confusing, I will rename it to SSSankeyDiagram